cmd/minikube/main.go

```diff
@@ -289,6 +289,7 @@ func propagateDockerContextToEnv() {
 	md, err := st.GetMetadata(currentContext)
 	if err != nil {
 		klog.Warningf("Unable to resolve the current Docker CLI context %q: %v", currentContext, err)
+		klog.Warningf("Try running `docker context use %s` to resolve the above error", currentContext)
```

Comment:
this is not always gonna be the case !
it is true that if they set a wrong context it will not be able to resolve the docker,  but it would say the error if the docker host doesnt exist...

I think it would worth it if minikube check for multiple contexts in the begining...where we Cache the docker system info we can cache how many Contexts it has and keep it for each minikube run so we if there are Multiple Context we can make a note of it in the Logs and make it a possible source of error

Reply:
never mind I think I mis-thought